features/step_definitions/test_steps.rb

```diff
@@ -204,7 +204,29 @@
   pending
 end
 
-Then(/^I select "([^"]*)" from the dropdown list with classname "([^"]*)"$/) do |name,classname|
-  browser.div(:class,classname).click
-  browser.select_list(:class => classname).div(:text => name).select
+
+Then(/^I click on the drop down box with div class "([^"]*)"$/) do |dropdown|
+  browser.div(:class,dropdown).click
+end
+
+
+Then(/^I select "([^"]*)" with index "([^"]*)" with div class "([^"]*)"$/) do |name, index, classname|
+
+end
+
+
+Then(/^I print the contents of dropdown box with the class name "([^"]*)"$/) do |classname|
+  browser.divs(:class => classname).each do |div|
+    puts div.text
+  end
+end
+
```

Comment:
Can we merge this one with the step def for `Then I print the prices` ?  Both have same commands, only different statements